jobs/adspert_python_dev.html

```diff
@@ -0,0 +1,40 @@
+---
+title: Python Software Developer
+company: Adspert (Bidmanagement GmbH)
+url: https://www.adspert.net/en/jobs/python-software-developer-mf/
+location: Berlin, Germany
+contract: permanent
+contact:
+    name: Karl Bartel
+    email: jobs@adspert.net
+created: !!timestamp '2017-05-15'
+tags:
+  - python
+  - sql
+---
+We are looking for a **Python Software Developer (m/f)**
```

Comment:
m/f meaning male/female? I don't know about Germany but in the UK this could open you up to legal action if, for example, any of your staff came out as transgender.